ibis/common/graph.py

```diff
@@ -340,9 +340,39 @@ def find(
         determined by a breadth-first search.
 
         """
-        nodes = Graph.from_bfs(self, filter=filter, context=context).nodes()
+        graph = Graph.from_bfs(self, filter=filter, context=context)
         finder = _coerce_finder(finder, context)
-        return [node for node in nodes if finder(node)]
+        return [node for node in graph.nodes() if finder(node)]
+
+    @experimental
+    def find_below(
```

Comment:
Are you planning on using this somewhere?

Reply:
@cpcloud added description